src/test/java/gatlingdemostore/SessionId.java

```diff
@@ -0,0 +1,18 @@
+package gatlingdemostore;
+
+import java.util.Random;
+
+public class SessionId {
+
+  private static final char[] CANDIDATES =
+      "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz".toCharArray();
+  private static final int SESSION_ID_LENGTH = 10;
+
+  static String random(Random random) {
```

Comment:
Same as above. This doesn't need a `Random` parameter. Just use ThreadLocalRandom internally.